layer/executables/layer_runtime.py

```diff
@@ -97,6 +104,50 @@ def _create_dataset(self, name: str, func: Callable[..., Any]) -> None:
         )
         data_catalog.store_dataset(func(), uuid.UUID(build_response.id.value))
 
+    def _create_model(self, name: str, func: Callable[..., Any]) -> None:
```

Comment:
The Context object created and set as active context to make logging and other stuff work because that's the way they figure out which function execution are happening. How are we going to solve that problem ?

Reply:
Haven't looked at it yet, but I feel it could be solved by not using the global state of the process.